examples/ex9p.cpp

```diff
@@ -178,7 +249,7 @@ int main(int argc, char *argv[])
    bool adios2 = false;
    bool binary = false;
    int vis_steps = 5;
-
+   PrecType prec_type = PrecType::AIR;
```

Comment:
Will the default value of `AIR` break any autotests? In other words, are there tests that use old versions of hypre? It seems safer to use the old version (`ILU`) as the default, although it is also good to encourage AIR if available. I don't have a strong preference here, but this is slightly concerning.

Reply:
I think it passed all the built in tests below except branch history and commit issues. That being said, if this is a concern another option is to set the default using the #if MFEM_HYPRE_VERSION pragma, either to AIR or ILU if hypre is not up to date.

Reply:
@bensworth I like the idea of using a `#if` to set the default according to the hypre version. This suggests to the user that AIR is recommended but will not cause trouble if it is not available.